src/scene/morph.js

```diff
@@ -221,6 +221,16 @@ class Morph extends RefCountedObject {
         this._targets.length = 0;
     }
 
+    /**
+     * @readonly
+     * @name Morph#numTargets
+     * @type {number}
+     * @description Read-only property that returns number of targets of this morph. Access targets using {@link Morph#getTarget}.
+     */
+    get numTargets() {
+        return this._targets ? this._targets.length : 0;
+    }
```

Comment:
We currently have `Morph#getTarget`. Any reason why we can't just have:

```javascript
get targets() {
    return this._targets;
}
```

Then, you could do:

```javascript
const numTargets = morph.targets.length;
```

Reply:
sounds good to me .. it changes the existing API but nobody would have used it without the 'count' property anyways .. so I've made the change. I'll update description as well.